roscala/src/main/scala/coop/rchain/roscala/ob/Tuple.scala

```diff
@@ -14,6 +15,101 @@ class Tuple(val value: Array[Ob]) extends Ob {
 
   def update(arg: Int, ob: Ob): Unit = value.update(arg, ob)
 
+  def accepts(msg: Ctxt): Boolean =
```

Comment:
This won't quite work because we have `Nil` as a different object.
Just do `this.value.exists(_.matches(msg))` here and in addition do:
```

object Nil extends Tuple(Array.empty[Ob]) {
  override def accepts(msg: Ctxt): Boolean = true
}
```

Reply:
done